packages/create-astro/src/actions/dependencies.ts

```diff
@@ -39,6 +44,45 @@ export async function dependencies(
 			},
 			while: () => install({ packageManager: ctx.packageManager, cwd: ctx.cwd }),
 		});
+
+		let add = ctx.add;
+		if (!add && !ctx.yes) {
+			let { adapter } = await ctx.prompt({
+				name: 'adapter',
+				type: 'select',
+				label: title('adapter'),
```

Comment:
The 7 char here seems to indent the CLI for me. Maybe we can rename this as "adapt"?

<img width="506" alt="image" src="https://github.com/user-attachments/assets/965dd381-3d30-4a4e-aa75-ef52379ef22a">

```suggestion
				label: title('adapt'),
```

Reply:
adpt perhaps? it matches template

Reply:
I'm fine either ways 👍